### ISSUE TITLE
Add the option to flag an image to the gallery

### ISSUE THREAD

**Issue by dreamfarer** [sopra-fs25-group-20/sopra-fs25-group-20-client#90]

**Comment by dreamfarer** [comment]
This optional requirement cannot be implemented due to time constraints. Previously unplanned usability and accessibility features take priority in order to provide a polished experience.
